### PR TITLE
imp(cli): Add client, cmd and version packages

### PR DESCRIPTION
**Pull request by MalteHerrmann**
**NOTE**: This can be merged after #22.

----

- **add client, cmd and version packages**

### CODE REVIEW

File: client/debug/debug.go [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+	}
+}
+
+func AddrCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "addr [address]",
+		Short: "Convert an address between hex and bech32",
+		Long:  "Convert an address between hex encoding and bech32.",
+		Example: fmt.Sprintf(
+			`$ %s debug addr evmos1qqqqhe5pnaq5qq39wqkn957aydnrm45sdn8583
```
Comment (0xstepit) [inline client/debug/debug.go:85]: nit: os1s....

File: client/debug/debug.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+				hexAddrString := common.BytesToAddress(hexAddr).String()
+
+				cmd.Println("Address bytes:", hexAddr)
+				cmd.Printf("Address hex: %s\n", hexAddrString)
+			}
+			return nil
+		},
+	}
+
+	cmd.Flags().String(flagPrefix, "", "Bech32 encoded account prefix, for example evmos, evmosvaloper")
```
Comment (0xstepit) [inline client/debug/debug.go:130]: ditto

File: client/debug/debug.go [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+		},
+	}
+}
+
+// LegacyEIP712Cmd outputs types of legacy EIP712 typed data
+func LegacyEIP712Cmd() *cobra.Command {
+	return &cobra.Command{
+		Use:     "legacy-eip712 [file]",
+		Short:   "Output types of legacy eip712 typed data according to the given transaction",
+		Example: fmt.Sprintf(`$ %s debug legacy-eip712 tx.json --chain-id evmosd_9000-1`, version.AppName),
```
Comment (0xstepit) [inline client/debug/debug.go:165]: ditto

File: client/debug/debug.go [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/evmos/os/ethereum/eip712"
+	evmos "github.com/evmos/os/types"
```
Comment (0xstepit) [inline client/debug/debug.go:14]:
```suggestion
	os "github.com/evmos/os/types"
```

**Review by 0xstepit — COMMENTED** [review]
Is client testnet.go file not required?

**Comment by MalteHerrmann** [comment]
> Is client testnet.go file not required?
That contains the outdated and not maintained `evmosd testnet` command, which is not functional since forever AFAIK - so I didn't want to port it to the new evmOS repo

**Review by 0xstepit — APPROVED** [review]

**Review by ramacarlucho — APPROVED** [review]

**Pull request merged**